unity/Assets/Plugins/Countly/CountlyManager.cs

```diff
@@ -195,9 +196,15 @@ protected void EndSession(long duration)
 
       builder.Append("&session_duration=");
       AppendConnectionData(builder, duration.ToString());
+			Log ("Requesting session end");
+		try {
+				WebRequest www = WebRequest.Create(appHost + "/i?" +builder.ToString());
+				www.GetResponse().Close();
 
-      ConnectionQueue.Enqueue(builder.ToString());
```

Comment:
what was wrong with using this connection queue? You could still make the WebRequest inside _ProcessConnectionQueue like I do with the WWW.